angular/src/components/export.component.ts

```diff
@@ -21,22 +23,34 @@ export class ExportComponent implements OnInit {
     @Output() onSaved = new EventEmitter();
 
     formPromise: Promise<string>;
-    masterPassword: string;
-    format: 'json' | 'encrypted_json' | 'csv' = 'json';
-    showPassword = false;
     disabledByPolicy: boolean = false;
 
+    exportForm = this.fb.group({
+        format: ['json'],
+        secret: [''],
+    });
```

Comment:
Me when somebody refactor an existing form to use reactive forms
![giphy](https://user-images.githubusercontent.com/15897251/140816138-ae07acb8-a7bc-4914-ba74-195d6e127aeb.gif)

Reply:
blame @Hinton ... but he's converting me